app/web/features/search/SearchResultsList.tsx

```diff
@@ -119,21 +119,21 @@ export default function SearchResultsList({
   const hasAtLeastOnePageResults =
     results && results?.pages[0]?.resultsList?.length !== 0;
 
-  let resultsList: any = results?.pages
+  let resultsList = results?.pages
     .flatMap((page) => page.resultsList)
     .filter((result) => result.user);
 
   let wasResultFound = false;
 
   // @TODO - Can make this one variable by using a find function, resultsList.find, etc.
-  resultsList?.map((value: any) => {
+  resultsList?.map((value) => {
     if (value.user?.userId === selectedResult?.userId) {
       wasResultFound = true;
     }
   });
 
   if (!wasResultFound && selectedUserData.data) {
-    resultsList = [{ user: selectedUserData.data }];
+    resultsList = [{ user: selectedUserData.data, rank: 0, snippet: "" }];
```

Comment:
Do we have any filters right now that take `rank` or `snippet` into account? I was also wondering this and if it could sort someone wrong due to that.

Reply:
I think should be fine since this only happens when the result isn't on the list, so we are displaying only one result (basically the first time map is loaded all the users are shown, but due pagination we show only a subset in the list (only on desktop)

this shows the user when the user clicks on it on the map, so list would be only 1 result, so I don't think the hardcoded rank will be a problem :)